openshift3/persistent-postgresql.json

```diff
@@ -256,7 +171,8 @@
                         "containers": [
                             {
                                 "name": "weblate",
-                                "image": "${APPLICATION_NAME}-img",
+                                "image": "weblate/weblate:${WEBLATE_VERSION}",
+                                "imagePullPolicy": "IfNotPresent",
```

Comment:
This will lead to never updating if somebody chooses `latest`. I have no clue how common that approach is on OpenShift though, so it might be actually okay :-).

Reply:
actually, you are right, this is not really wanted. Will change it to `Always`, makes more sense.